doc/files.am

```diff
@@ -4270,16 +4280,23 @@ html_files_relative_from_locale_dir = \
 	html/_static/switcher.json \
 	html/_static/underscore-1.13.1.js \
 	html/_static/underscore.js \
-	html/_static/vendor/fontawesome/6.1.2/LICENSE.txt \
-	html/_static/vendor/fontawesome/6.1.2/css/all.min.css \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.ttf \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-brands-400.woff2 \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.ttf \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-regular-400.woff2 \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.ttf \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-solid-900.woff2 \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-v4compatibility.ttf \
-	html/_static/vendor/fontawesome/6.1.2/webfonts/fa-v4compatibility.woff2 \
+	html/_static/vendor/fontawesome/5.13.0/LICENSE.txt \
```

Comment:
Did you use old Sphinx?

Reply:
https://github.com/groonga/groonga/runs/8300752300?check_suite_focus=true#step:8:16688

> cp: cannot stat './html/_static/vendor/fontawesome/5.13.0/LICENSE.txt': No such file or directory

Reply:
Hmm, I am using the latest Sphinx.
The latest version is 5.1.1: https://pypi.org/project/Sphinx/


```
$ pip show sphinx
Name: Sphinx
Version: 5.1.1
Summary: Python documentation generator
Home-page: https://www.sphinx-doc.org/
Author: Georg Brandl
Author-email: georg@python.org
License: BSD
Location: /home/t-hashida/.local/lib/python3.8/site-packages
Requires: alabaster, babel, docutils, imagesize, importlib-metadata, Jinja2, packaging, Pygments, requests, snowballstemmer, sphinxcontrib-applehelp, sphinxcontrib-devhelp, sphinxcontrib-htmlhelp, sphinxcontrib-jsmath, sphinxcontrib-qthelp, sphinxcontrib-serializinghtml
Required-by: pydata-sphinx-theme
```

Reply:
pydata_sphinx_theme?

Reply:
I used `pydata_sphinx_theme` v0.9.0, but the latest is v0.10.1.

I have updated `pydata_sphinx_theme` and re-execute `make update-files`.